Makefile

```diff
@@ -0,0 +1,91 @@
+all: jenkins prow mungegithub projects
+.PHONY: all
+
+jenkins:
+.PHONY: jenkins
+
+prow: prow-crd prow-config prow-images hook plank jenkins-operator deck horologium splice sinker commenter
+.PHONY: prow
+
+prow-crd:
+	oc apply -f cluster/ci/config/prow/prow_crd.yaml
+.PHONY: prow-crd
+
+prow-config:
+	oc apply -f cluster/ci/config/prow/config.yaml cluster/ci/config/prow/plugins.yaml
+.PHONY: prow-config
+
+prow-images:
+	oc process -f cluster/ci/config/prow/prow_images.yaml | oc apply -f -
+.PHONY: prow-images
+
+hook:
+	# secrets
```

Comment:
Let's create/update the secrets in a separate target?

Reply:
ack